.travis.yml

```diff
@@ -34,7 +32,7 @@ jobs:
       script:
         - docker run --name "openhabian-test" --privileged -d openhabian/rpi-openhabian
         - docker exec -it "openhabian-test" bash -c \
-            'VIRT=$(sudo virt-what); echo -e "\n\e[34mTesting openhabian install on ${VIRT} ...\n"; ./build.bash local-test && mv ~/.profile ~/.bash_profile && /etc/rc.local; OUTPUT=$?; if [ "${VIRT}" != "native HW" ]; then echo -e "\n\e[32mDid not test openhabian install on native HW, ignoring test result.\n"; exit 0; fi'
+            'VIRT=$(sudo virt-what); echo -e "\n\e[34mTesting openhabian install on ${VIRT} ...\n"; ./build.bash local-test && mv ~/.profile ~/.bash_profile && /etc/rc.local; OUTPUT=$?; if [ "${VIRT}" != "native HW" ]; then echo -e "\n\e[32mDid not test openhabian install on native HW, ignoring test result.\n"; fi'
```

Comment:
don't think thil will work... the if will always eat up the $? if the toll to rc.local, i.e. the firsttime install. IMHO always counted as ok, regardless of HW. Or did I overlook what $OUTPUT is doing here?

I would suggest something which is more clear
`'VIRT=$(sudo virt-what); echo -e "\n\e[34mTesting openhabian install on ${VIRT} ...\n";\
if [ "${VIRT}" != "native HW" ]; then 
  echo -e "\n\e[32mNo running on native HW, ignoring test result from following test:\n";\
   (./build.bash local-test && mv ~/.profile ~/.bash_profile && /etc/rc.local;) || true
else\
   ./build.bash local-test && mv ~/.profile ~/.bash_profile && /etc/rc.local\
fi`
(untested)

Reply:
ok. The idea was to assign the test result to $OUTPUT but I think the line was incomplete so changed to your proprosal as it's clearer